elasticpypi/config.py

```diff
@@ -1,7 +1,26 @@
-import json
 import os
 
+WATCHED=[
+    'SERVICE',
+    'STAGE',
+    'BUCKET',
+    'TABLE',
+    'USERNAME',
+    'PASSWORD',
+    'OVERWRITE'
+]
+
+TRUTHY = ['TRUE', 'true', 1]
+FALSEY = ['FALSE', 'false', 0]
+
+# load config
+config = {k.lower(): v for k, v in os.environ.iteritems() if k in WATCHED}
+
+# check config
+for k, v in config.iteritems():
+    # assume env vars are always stringy
+    if v in TRUTHY:
+        config[k] = True
```

Comment:
I would also expect it to be user's responsibility to correctly key the values. If there is a boolean config option, we should give the default correctly and have docs that explain that it is taken as is.